monri/src/main/java/com/monri/android/activity/ConfirmPaymentActivity.java

```diff
@@ -1,24 +1,28 @@
 package com.monri.android.activity;
 
+
 import android.app.Activity;
 import android.content.Context;
 import android.content.Intent;
-import android.content.SharedPreferences;
 import android.os.Bundle;
-import android.preference.PreferenceManager;
+import android.os.Parcel;
+import android.os.Parcelable;
 import android.widget.ProgressBar;
 
-import androidx.appcompat.app.AppCompatActivity;
+import androidx.activity.ComponentActivity;
+import androidx.activity.result.ActivityResultCaller;
+import androidx.annotation.Nullable;
 
 import com.monri.android.BuildConfig;
 import com.monri.android.Monri;
 import com.monri.android.MonriUtil;
 import com.monri.android.R;
 import com.monri.android.model.ConfirmPaymentParams;
 import com.monri.android.model.MonriApiOptions;
+import com.monri.android.model.PaymentResult;
 import com.monri.android.three_ds1.auth.PaymentAuthWebView;
 
-public class ConfirmPaymentActivity extends Activity {
+public class ConfirmPaymentActivity extends ComponentActivity {
```

Comment:
@adnanomerovic this is backwards compatible? I.e., it will work if you include our SDK - without issues on client's side?

Reply:
Backwards compatible (it's part of androidx.appcompat:appcompat, lib in build gradle), this new way require ComponentActivity otherwise it won't work.